<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMessageDialog</class>
 <widget class="QDialog" name="NewMessageDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Message</string>
  </property>
  <layout class="QVBoxLayout" name="newMessageDialogVLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="QFrame" name="topFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="topFrameVLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="bodyFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="bodyFrameVLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>9</number>
         </property>
         <item>
          <layout class="QGridLayout" name="protocolGridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="protocolLabel">
             <property name="text">
              <string>Protocol Name  </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="protocolComboBox">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <property name="maxVisibleItems">
              <number>20</number>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="protocolHorizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>88</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="messageHLayout">
           <item>
            <widget class="QLabel" name="messageLabel">
             <property name="text">
              <string>Message Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="messageLineEdit">
             <property name="placeholderText">
              <string>Message1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="descriptionHLayout">
           <item>
            <widget class="QLabel" name="descriptionLabel">
             <property name="text">
              <string>Description      </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="descriptionLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="detailedHLayout">
           <item>
            <layout class="QVBoxLayout" name="detailedVLayout">
             <item>
              <widget class="QLabel" name="detailedLabel">
               <property name="text">
                <string>Detailed Desc </string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>68</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="detailedTextEdit">
             <property name="tabChangesFocus">
              <bool>true</bool>
             </property>
             <property name="overwriteMode">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="buttonFrame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="buttonFrameVLayout">
         <property name="margin">
          <number>9</number>
         </property>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="buttonHLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <spacer name="buttonHorizondalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>178</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="createButton">
             <property name="text">
              <string>Create</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cancelButton">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>protocolComboBox</tabstop>
  <tabstop>messageLineEdit</tabstop>
  <tabstop>descriptionLineEdit</tabstop>
  <tabstop>detailedTextEdit</tabstop>
  <tabstop>createButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
